odoo/addons/base/security/ir.model.access.csv

```diff
@@ -17,8 +17,9 @@
 "access_ir_model_data_user","ir_model_data user","model_ir_model_data",base.group_user,0,0,0,0
 "access_ir_model_fields_user","ir_model_fields all","model_ir_model_fields",base.group_user,0,0,0,0
 "access_ir_model_fields_selection_user","ir_model_fields_selection all","model_ir_model_fields_selection",base.group_user,0,0,0,0
-"access_ir_module_category_group_user","ir_module_category group_user","model_ir_module_category","group_erp_manager",1,0,0,0
-"access_ir_module_module_group_user","ir_module_module group_user","model_ir_module_module","group_system",1,1,1,1
+"access_ir_module_category_group_user","ir_module_category group_user","model_ir_module_category","group_user",1,0,0,0
+"access_ir_module_module_group_user","ir_module_module group_user","model_ir_module_module","group_user",1,0,0,0
```

Comment:
hey no hein dis donc

Reply:
if you have moved the logic to another module, you can also move the security rule there